File: Asahel-code/fizz_buzz.js
// Declaration of an empty array
const results = [];

// loop through 1 to 100
for (let i = 1; i <= 100; i++) {
     // Check if there is a number that is multiple of both 3 and 5 and return FizzBuzz
    if (i % 5 === 0 && i % 3 === 0) {
        results.push("FizzBuzz");
    }
    // Check if there is a number that is multiple of 3 and return Fizz
    else if (i % 3 === 0) {
        results.push("Fizz");
    }
    // Check if there is a number that is multiple of 5 and return Buzz
    else if (i % 5 === 0) {
        results.push("Buzz");
    }
    // Return the number if it is not a multple of 3 and 5 
    else {
        results.push(i);
    }
}

console.log(results);

File: Kmtengo/FizzBuzz.py
# Loop through numbers from 1 to 100
for i in range(1, 101):
    # Check if the number is divisible by both 3 and 5
    if i % 3 == 0 and i % 5 == 0:
        print("FizzBuzz")
    # Check if the number is divisible by 3
    elif i % 3 == 0:
        print("Fizz")
    # Check if the number is divisible by 5
    elif i % 5 == 0:
        print("Buzz")
    # If the number is not divisible by 3 or 5, print the number
    else:
        print(i)
